packages/monitor-v2/src/monitor-polymarket/MonitorProposalsOrderBook.ts

```diff
@@ -54,19 +54,33 @@ export async function monitorTransactionsProposedOrderBook(
 
   // Filter out markets that do not have a proposal event.
   const marketsWithEventData: PolymarketWithEventData[] = marketsWithAncillary
-    .filter((market) =>
-      proposalEvents.find(
+    .filter((market) => {
+      if (
+        ![params.binaryAdapterAddress.toLowerCase(), params.ctfAdapterAddress.toLowerCase()].includes(
+          market.resolvedBy.toLowerCase()
+        ) &&
+        market.umaResolutionStatus.toLowerCase().includes("proposed")
+      )
+        logUnknownMarketAdapter(logger, {
```

Comment:
If the market is proposed and the market.resolvedBy is not the `binaryAdapterAddress` or `ctfAdapterAddress` the market will be filtered out so we want to notify to have it manually checked.